utils/telemetry/tracker.go

```diff
@@ -0,0 +1,58 @@
+package telemetry
+
+import (
+	"fmt"
+	"github.com/mitchellh/cli"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+var (
+	segmentWriteKey string //value will be injected at build-time
```

Comment:
how are we injecting this?

Reply:
in the `go build` command, using:

`-ldflags="-X 'github.com/10gen/realm-cli/utils/telemetry.segmentWriteKey=$SEGMENT_WRITE_KEY'"`

Reply:
should that be in the .evg.yml file then?

Reply:
indeed (i'll be making sure to do a more thorough review now that weve landed on an approach and get some tests eventually)... @jo-tse-mongo, if you want to get ahead of it here, you can check out the `go build...` command in `evg.yml` on `master` branch ... that should show you how we're trying to include these write keys (you also should be able to use that same env var)